src/plugins/data/public/search/aggs/buckets/filter.ts

```diff
@@ -30,6 +31,12 @@ export interface FilterBucketAggDependencies {
   getInternalStartServices: GetInternalStartServicesFn;
 }
 
+export interface AggParamsFilter {
+  field: string;
```

Comment:
I'm sure `field` must be required here, but for whatever reason I'm not finding where in the code this param is actually added to the filter type -- do you know?

Reply:
@lukeelmers interesting, but not sure that you are right here. Looks like it was my issue. I removed `field: string;` from `AggParamsFilter`

Also I've created: #65047 to discuss.